editor/create_dialog.cpp

```diff
@@ -160,8 +160,14 @@ bool CreateDialog::_should_hide_type(const String &p_type) const {
 
 		String script_path = ScriptServer::get_global_class_path(p_type);
 		if (script_path.begins_with("res://addons/")) {
-			if (!EditorNode::get_singleton()->is_addon_plugin_enabled(script_path.get_slicec('/', 3))) {
-				return true; // Plugin is not enabled.
+			int i = script_path.find("/", String("res://addons/").length());
```

Comment:
Could hardcode the value and explain it's the length of `res://addons/` in a comment, to avoid re-calculating it each time (unless we're confident compilers will optimize it).